jest.config.packages.js

```diff
@@ -22,7 +22,13 @@ module.exports = {
   collectCoverage: true,
 
   // An array of glob patterns indicating a set of files for which coverage information should be collected
-  collectCoverageFrom: ['./src/**/*.ts', '!./src/**/*.test-d.ts'],
+  collectCoverageFrom: [
+    './src/**/*.ts',
+    // Ignore typing test files
+    '!./src/**/*.test-d.ts',
+    // Ignore index of subdirectories
+    '!./src/**/*/index.ts',
```

Comment:
This is a bit new and could be done differently if we were using the top-level module in our tests I think. But IMO, having coverage on `index.ts` does not make sense with our current setup (those are only re-exports)

Reply:
Just reposting what we saind in our daily: testing the `index.ts` files make sense to ensure that we cover the public API. Sometimes in the past we did a release just to realise that we forgot to add an export to the `index.ts` file.

Reply:
I'll go ahead with the current PR and do this in another one (as an improvement)